src/components/dev/DevHud.js

```diff
@@ -81,17 +81,30 @@ const DevMenu = function ({ devProps, funcs, data, state }) {
 
 class DevHud extends Component {
 
-  state = {
-    hiderText:  `^ --- Hide --- ^`,
-    toHideText: `^ --- Hide --- ^`,
+  constructor (props) {
+    super(props);
+
+    var toggleContent = `Hide`;
+    if (props.devProps.hidden) {
+      toggleContent = `Show`;
+    }
+
+    this.state = { visibilityToggleContent: toggleContent };
```

Comment:
It's true. I guess I could use a ternary the way @ethanbb demonstrated. I could live with that.

Reply:
How about getting rid of the ( ) which aren't needed in the ternary?  Less to read feels easier on the eyes.

```
{ devProps.devHidden ? (
  `Show`
) : (
  `Hide`
) }

becomes 

{ 
  devProps.devHidden
  ? `Show`
  : `Hide`
}

or 

{ 
  devProps.devHidden ?
  `Show` :
  `Hide`
}
```

Reply:
I think we decided to try to make the multi-line ternaries we are using consistent, and when there are bigger things after `?` and `:` the parentheses do make things clearer (and help avoid an eslint conflict in some cases).

That being said, this could probably be a single-line ternary.

Reply:
Definitely agree that it can be beneficial when they're longer.  In this case, they don't serve any real purpose.  But you raise a good point that it might make more sense to shorten this to the following which is much more purdy.  :)

```
{ devProps.devHidden ? `Show` : `Hide` }
```

Reply:
I can change it to the one liner. One of the things I like about the parens is that it also makes it easier to distinguish what parts are what. In this case, it isn't that complicated so maybe it's not as bad. I always have trouble spotting the `:` in there all tiny and mixed in with everything else. Another thing with the multi-line ones is that it looks more like an `if`.